include/beast/http/detail/rfc7230.hpp

```diff
@@ -318,20 +324,23 @@ increment()
     {
         ++it;
         if(it == end)
-            return err();
+            break;
         if(! detail::is_tchar(*it))
             break;
     }
```

Comment:
@ximinez  Good suggestion, as long as `it` is incremented before the call.

Or

while(++it != end && is_tchar(*it));